Improve docs metadata (#405)
o1-labs/docs2: docs/zkapps/advanced-snarkyjs/time-locked-accounts.mdx

---
title: Time-Locked Accounts
hide_title: true

> description: This guide dives into the feature of time-locking in Mina, explaining how you can use it to implement vesting schedules. It further provides practical examples of setting up a time-lock for MINA tokens using SnarkyJS.

Reviewer: ```suggestion
description: This guide dives into the feature of time-locking in Mina to implement vesting schedules and practical examples of setting up a time-lock for MINA tokens using SnarkyJS.
```
